spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/SparkCatalog.java

```diff
@@ -921,13 +926,19 @@ private Table loadFromPathIdentifier(PathIdentifier ident) {
     String branch = null;
     String tag = null;
     boolean isChangelog = false;
+    boolean isRewrite = false;
 
     for (String meta : parsed.second()) {
       if (meta.equalsIgnoreCase(SparkChangelogTable.TABLE_NAME)) {
         isChangelog = true;
         continue;
       }
 
+      if (REWRITE.equals(meta)) {
+        isRewrite = true;
```

Comment:
Looks like I missed the `equalsIgnoreCase` here on the original PR, it's inconsequential but it'd be nice to have a comparison consistency like the changelog table. I've made a note to follow up on this cleanup but for now to keep the backport as clean as possible I'll keep this as is.